pages/_app.js

```diff
@@ -109,6 +109,22 @@ export default function App({ Component, pageProps }) {
     flashcardsMutate();
   }
 
+  async function handleUpdateCollection(_id, data) {
```

Comment:
See my previous comments.

```suggestion
  async function handleUpdateCollection(data) {
```